src/components/Avatar/Avatar.types.ts

```diff
@@ -5,7 +5,68 @@ import { ListProps } from '../List';
 import { TooltipProps } from '../Tooltip';
 import { OcBaseProps } from '../OcBase';
 
-interface BaseAvatarProps extends OcBaseProps<HTMLSpanElement> {
+export type StatusItemsMap = {
+  [key in StatusItemsPosition]?: StatusItemsProps;
+};
+
+export enum StatusItemsPosition {
+  Top = 'top',
+  Bottom = 'bottom',
+  Left = 'left',
+  Right = 'right',
+  TopRight = 'topright',
```

Comment:
nit: make all these camelcase
```suggestion
  TopRight = 'topRight',
```